### PR TITLE
Abductor balance changes

### PR DESCRIPTION
**Pull request by PowerfulBacon**
Includes https://github.com/BeeStation/BeeStation-Hornet/pull/7775 that PR should be automatically merged if this one is merged, however I expect 7775 to be merged first

## About The Pull Request

 - Abductor silences now acts like a radio jammer, rather than an active use item.
 - Abductor baton can now only stun/sleep a single target at a time. If you attack another target while you have a target stunned already, they will have their stun removed.
 - Abductors can no longer teleport anywhere that a specimin is viewing. You can warp near 

## Why It's Good For The Game

Alters the balance of abductors to focus specifically on their powerful nature in stealthy scenarios, abductors can no longer go 1v6 aganist the entirity of security with their baton capable of stunning a target instantly, as the baton can now only stun a single target at a time. Abductors can no longer teleport on top of people, as this is guaranteed to blow their cover to the specimin.
This weakens the ability of abductors to do non-abducting behaviours and forces them to think, as they can be overwhelmed very easilly after losing the ability to fight more than 1 target at a time with the baton.

## Testing Photographs and Procedure

image
image
image
image

## Changelog
:cl:
balance: Abductor silencer is now a passive radio jammer.
balance: Abductor baton can only incapacitate a single target at a time. Attacking multiple targets will result in the previous target waking up.
balance: Abductors can no longer teleport into locations that are being viewed by valid specimins.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Epic rust_g fail

File: code/modules/antagonists/abductor/machinery/camera.dm [hunk 1]
```diff
@@ -69,12 +69,26 @@
...
+		//Not a specimin
+		if(temp)
+			continue
+		//No heart, not considered a specimin
+		if (!specimin.getorganslot(ORGAN_SLOT_HEART))
+			continue
+		//Technically a specimin, however we should avoid meta tactics
+		if (!specimin.client)
+			continue
+		to_chat(owner, "<span class='warning'>You cannot warp to this location, an unprocessed specimin might spot you, tampering with the experiment!</span>")
```
Comment (PowerfulBacon) [inline code/modules/antagonists/abductor/machinery/camera.dm]:
```suggestion
		to_chat(owner, "<span class='warning'>You cannot warp to this location, an unprocessed specimen might spot you, tampering with the experiment!</span>")
```

File: code/modules/antagonists/abductor/machinery/camera.dm [hunk 2]
```diff
@@ -101,12 +115,26 @@
...
```
Comment (PowerfulBacon) [inline code/modules/antagonists/abductor/machinery/camera.dm]:
```suggestion
		to_chat(owner, "<span class='warning'>You cannot warp to this location, an unprocessed specimen might spot you, tampering with the experiment!</span>")
```

File: code/modules/antagonists/abductor/equipment/abduction_gear.dm [hunk 3]
```diff
@@ -459,6 +432,21 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 	to_chat(usr, "<span class='notice'>You switch the baton to [txt] mode.</span>")
 	update_icon()
 
+/obj/item/abductor/baton/examine(mob/user)
+	. = ..()
+	if (!isabductor(user))
+		return
+	. += "The device emits active neuro-impulses to stun only a single target at a time."
```
Comment (PowerfulBacon) [inline code/modules/antagonists/abductor/equipment/abduction_gear.dm]:
```suggestion
	. += "The device emits active neuro-impulses capable of stunning up to a single target at a time."
```

**Comment by Kratairo** [comment]
I ded pls nerf vibes

**Comment by Fronsis** [comment]
Hmmm i feel like it would be very strange for the person who just recently got stun/sleep'd to randomly get up full 100% ready to fight back if the abductor decides to stun another person, i believe it would be more balanced if the first person stays temporally in a stutter-like state making them drop stuff or wake up with stam crit.

Though overall... it will make it a little hard to secure experiments, i would see it limited to two stun/cuff'd characters so at least you could have a little more chance, on a highpop it's very unlikely to find people isolated limiting to two people could allow attempts at places like Xenolab otherwise it's gonna feel like watching a monitor simulator + the Agent player playing with their tumbs

I'm all up to remove the concept of Abductors 1v6 people, that's why i suggested a SOP/Antagonist conduct to prevent that type of crazy 1v6 abusing insta-stun baton rather than nerfing their operating tools that makes them be able to do their mission

**Comment by PowerfulBacon** [comment]
SOP/rule changes to fix balancing and design issues is not a good idea and will never work out. It will make it more difficult for abductors in the sense that they can't run into a room with 3-4 people and stun all of them, only to take one (they should only be able to take 1 at a time anyway), however that is the intention of these changes. Abductors will need to put in more work into finding targets which are either alone, or moving around and will pose more of a challenge while still providing the teleportation, recall and powerful baton that makes it an accessible antag.

**Review by itsmeow — DISMISSED** [review]

**Review by itsmeow — APPROVED** [review]

**Pull request merged**